runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -181,7 +188,10 @@ public VersionLockedObject(Supplier<T> newObjectFn,
                 log.trace("Access [{}] Direct (optimistic-read) access at {}",
                         this, getVersionUnsafe());
                     R ret = accessFunction.apply(object);
+
+                    long versionForCorrectness = getVersionUnsafe();
                     if (lock.validate(ts)) {
+                        correctnessLogger.trace("Version, {}", versionForCorrectness);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Version, {}" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)